cmd/builder/internal/builder/main_test.go

```diff
@@ -80,6 +80,43 @@ func TestGenerateAndCompileDefault(t *testing.T) {
 	time.Sleep(1 * time.Second)
 }
 
+func TestGenerateAndCompileWithLDFlags(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.Skip("skipping the test on Windows, see https://github.com/open-telemetry/opentelemetry-collector/issues/5403")
```

Comment:
This issue has been closed, is this skip still needed?

Reply:
I'm not sure - I'll try removing it.  It's in other main_test.go tests so I will be removing it there as well.

Reply:
It still failed so I reverted.  It seems like we still need to skip these tests on windows.